images/federation-v2/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+FROM ubuntu:14.04
```

Comment:
(question) Any reason for using `14.04`, and not the latest stable `16.04`?

Reply:
This is the exact dockerfile that is being used in the current apiserver-builder command that builds the federation-v2 container currently (https://github.com/kubernetes-incubator/apiserver-builder/blob/master/cmd/apiserver-boot/boot/build/build_container.go#L87). I have no issue changing it to 16.04 but I'm not sure if it would break something.

Reply:
As in my comment, posted it as a question, as was simply curios; seems like the right thing to do for now.